src/NuGet.Clients/PackageManagement.UI/Xamls/PackageItemControl.xaml.cs

```diff
@@ -14,6 +15,57 @@ public partial class PackageItemControl : UserControl
         public PackageItemControl()
         {
             InitializeComponent();
+            this.DataContextChanged += PackageItemControl_DataContextChanged;
```

Comment:
This doesn't look like a weak even? Otherwise how do you deal with memory leak here